### PR TITLE
Add Debug object and added callstack method to deprecated function wa…

### PR DESCRIPTION
**Pull request by Sleepyz4life**
…rnings

### CODE REVIEW

File: wordpress-objects/Clarkson_Object.php [hunk 1]
```diff
@@ -17,6 +17,7 @@ public function __construct( $post ) {
 			$this->_post = $post;
 		}else{
 			trigger_error("Deprecated __construct called with an ID. Use `::get($post)` instead.", E_USER_DEPRECATED);
+            Clarkson_Debug::callStack();
```
Comment (jmslbam) [inline wordpress-objects/Clarkson_Object.php]: If we would like to add the debugging to a different plugin then this could become an `apply_filter`?

Comment (jmslbam) [inline wordpress-objects/Clarkson_Object.php]: Or better yet, `do_action('clarkson_debug_do_stacktrace')` of `callstack`

Comment (Sleepyz4life) [inline wordpress-objects/Clarkson_Object.php]: I'll look into this either today or this week!

**Comment by jmslbam** [comment]
@Sleepyz4life Your debug started working properly when reinstalled VVV?

**Pull request closed without merging**